error.go

```diff
@@ -15,6 +15,8 @@ const (
 	CancelError ErrorType = "cancel"
 	// ConfigError is for errors reading/saving config
 	ConfigError ErrorType = "config"
+	// ConfigError is for when the GUI is active
+	GUIBusyError ErrorType = "guiBusy"
```

Comment:
Now this dude's unused

Reply:
It's used in IsGuiBusy() to test for this case in report()

Reply:
oh yeah my bad